src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -814,25 +823,23 @@ public void SendPing_LocaleEnvVarsMustBeIgnored(AddressFamily addressFamily, str
         public void SendPingAsync_LocaleEnvVarsMustBeIgnored(AddressFamily addressFamily, string envVar_LANG, string envVar_LC_MESSAGES, string envVar_LC_ALL)
         {
             IPAddress localIpAddress = TestSettings.GetLocalIPAddress(addressFamily);
-            if (localIpAddress == null)
-            {
-                // No local address for given address family.
-                return;
-            }
-
-            var remoteInvokeStartInfo = new ProcessStartInfo();
 
-            remoteInvokeStartInfo.EnvironmentVariables["LANG"] = envVar_LANG;
-            remoteInvokeStartInfo.EnvironmentVariables["LC_MESSAGES"] = envVar_LC_MESSAGES;
-            remoteInvokeStartInfo.EnvironmentVariables["LC_ALL"] = envVar_LC_ALL;
+            var remoteInvokeStartInfo = new ProcessStartInfo {
+                EnvironmentVariables =
+                {
+                    ["LANG"] = envVar_LANG,
+                    ["LC_MESSAGES"] = envVar_LC_MESSAGES,
+                    ["LC_ALL"] = envVar_LC_ALL
+                }
+            };
 
             RemoteExecutor.Invoke(async address =>
             {
                 await SendBatchPingAsync(
                     (ping) => ping.SendPingAsync(address),
                     (pingReply) =>
                     {
-                        PingResultValidator(pingReply, new IPAddress[] { IPAddress.Parse(address) }, null);
+                        PingResultValidator(pingReply, new[] { IPAddress.Parse(address) }, null);
```

Comment:
I'm not sure whether this change breaks coding style rule #10. At a minimum it needs clarification. @bartonjs thoughts/preferences?

Reply:
You're right. It breaks coding style rule 10.